wallet/src/Pos/Wallet/Web/Account.hs

```diff
@@ -132,14 +132,13 @@ generateUnique desc RandomSeed generator isDuplicate = loop (100 :: Int)
                       \you are approaching the limit") desc
     loop i = do
         rand  <- liftIO randomIO
-        value <- generator rand
-        bad   <- orM
-            [ isDuplicate rand value
-            , pure $ isHardened rand  -- using hardened keys only for now
-            ]
-        if bad
-            then loop (i - 1)
-            else return value
+        if isHardened rand then
+            loop (i - 1) -- use non-hardened keys only for now
```

Comment:
@gromakovsky We now need to backport the bugfixes introduced by #1991 as we did for the [PR targeting release/1.0.4](https://github.com/input-output-hk/cardano-sl/pull/2195#discussion_r159231021), as Ilya [did already](https://github.com/input-output-hk/cardano-sl/pull/2195#issuecomment-354787039) 😉